<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCalibrateTrack</class>
 <widget class="QWidget" name="QCalibrateTrack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">QCalibrateTrack</string>
  </property>
  <widget class="QComboBox" name="calibrate_comboBox">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>120</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="project_comboBox">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>120</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="CScatterPlotClass" name="TracckCurve" native="true">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>179</y>
     <width>1034</width>
     <height>606</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>1099</x>
     <y>100</y>
     <width>771</width>
     <height>690</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>45</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>1757</x>
     <y>817</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1617</x>
     <y>817</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>177</y>
     <width>120</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="Auto_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>自动</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Manual_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>66</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>手动</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="maxy_Edit">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>327</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="miny_Edit">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>404</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="Start_Edit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>120</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="End_Edit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>120</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>1045</width>
     <height>690</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">page_block</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>1099</x>
     <y>789</y>
     <width>771</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v">
   <property name="geometry">
    <rect>
     <x>1099</x>
     <y>100</y>
     <width>1</width>
     <height>689</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>1869</x>
     <y>100</y>
     <width>1</width>
     <height>689</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>1099</x>
     <y>100</y>
     <width>771</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>105</y>
     <width>1035</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="bks" stdset="0">
    <string notr="true">gray</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>367</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>最小吸光度</string>
   </property>
   <property name="fonts" stdset="0">
    <string notr="true">16</string>
   </property>
   <property name="colors" stdset="0">
    <string notr="true">disabled</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>290</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>最大吸光度</string>
   </property>
   <property name="fonts" stdset="0">
    <string notr="true">16</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>120</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>项目</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>120</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>校准品</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>54</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>校准日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>120</y>
     <width>20</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>calibrate_comboBox</zorder>
  <zorder>project_comboBox</zorder>
  <zorder>TracckCurve</zorder>
  <zorder>tableView</zorder>
  <zorder>close_btn</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>Start_Edit</zorder>
  <zorder>End_Edit</zorder>
  <zorder>groupBox</zorder>
  <zorder>maxy_Edit</zorder>
  <zorder>miny_Edit</zorder>
  <zorder>line_h_2</zorder>
  <zorder>line_v</zorder>
  <zorder>line_v_2</zorder>
  <zorder>line_h</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CScatterPlotClass</class>
   <extends>QWidget</extends>
   <header>src/leonis/curveDraw/ScatterPlotClass.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
